debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/unbuffered/UnbufferedLogMinerStreamingChangeEventSource.java

```diff
@@ -252,15 +226,13 @@ private PreparedStatement createQueryStatement() throws SQLException {
     /**
      * Processes the Oracle LogMiner data between the specified bounds.
      *
-     * @param context the change event source context, should not be {@code null}
      * @param minCommitScn mining range lower bounds SCN, should not be {@code null}
      * @return the next iteration's lower bounds SCN, never {@code null}
      * @throws SQLException if a database exception occurred
      * @throws InterruptedException if the thread is interrupted
      */
-    private Scn process(ChangeEventSource.ChangeEventSourceContext context, Scn minCommitScn)
-            throws SQLException, InterruptedException {
-        counters.reset();
+    private Scn process(Scn minCommitScn) throws SQLException, InterruptedException {
```

Comment:
The body of method also seems very similar to the buffered vesrion - save for initial query building.